C-Sharp/other/Largest_Kth_Element.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
```

Comment:
Add a beginner friendly pitch here

Reply:
@HarshCasper  Done, You can review it again now